tests/runtime/main.rs

```diff
@@ -586,75 +587,16 @@ fn tests(name: &str, dir_name: &str) -> Result<Vec<PathBuf>> {
                 panic!("failed to compile");
             }
 
-            let out_wasm = out_wasm.join("AppBundle").join(assembly_name);
+            let out_wasm = out_dir
+                .join("bin")
+                .join(configuration)
+                .join("net9.0")
```

Comment:
`tmf`

Reply:
that is only one of the issues here, build doesn't respect the project name, and the publish doesn't respect the output directory.  @maraf we should discuss what options we have here